### PR TITLE
Draft JOSS Paper

### PR DESCRIPTION
**Pull request by JosiahParry**
This PR adds a `paper.md` to be used for a publication in JOSS.

### CODE REVIEW

**Comment by Ilia-Kosenkov** [comment]
Ok this will sound stupid, but we have a draft paper and I do not remember if we published it. There is a hidden repo in the organization, let me check if I can do something about it

**Comment by CGMossa** [comment]
I have emailed everyone that is an author on extendr to collaborate on an extendr paper. I believe this conversation is outside of the scope of everyday-maintenance, and thus should be held separately.

@yutannihilation: I don't have your e-mail address. I have sent you a message about it in twitter/X, and I've asked the other collaborators to forward it to you, if they have your e-mail.

If you haven't received an e-mail, then it is a slip up from me, and please do write here or somewhere to me, and I'll forward it to you!

**Comment by yutannihilation** [comment]
You can find my email address here. But, I'm not interested in the paper, so I think I won't join the discussion anyway, sorry. I hope it goes well.

https://github.com/yutannihilation/string2path/blob/efa5f63adeb95a7ffd27959ad5b7a8a46474a2f7/DESCRIPTION#L8

File: paper.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+https://www.r-consortium.org/all-projects/awarded-projects/2021-group-1#extendr+-+rust+extensions+for+r.
+
+related software Rcpp, cpp11,
+
+## 
+
+## Adoption in the R ecosystem
+
+-   rpolars
```
Comment (eitsupi) [inline paper.md:122]:
```suggestion
-   polars
```

**Pull request closed without merging**